### DISCUSSION TITLE
Input and function versus input and output

### DISCUSSION THREAD

**Discussion by ghost** [xyflow/xyflow#2992] [Polls]
I believe react-flow is a graph based interface, which means, nodes have input and output. But with reference to the latest computing technology, instead of input providing to output, we need to brainstorm for a solution for input plus output is equal to a function.

**Background:**
Input plus function is equal to output is the past tech trend, conventional programming where producers are programmers and consumers are users.

input and output is equal to function is the current trend, Artifical intelligence programming where the consumers are the programmers. 

**Conclusiotn**

With that said, I wanted to start a thread to evolve react-flow or the node based techniques to the modern computing technique.

**PS** 
please note that I'm not promoting AI trends, I believe conventional or the traditional techniques will never outpace new, but its good for the project's vision to be prepared for the next gen.

**Comment by moklick** [comment]
React Flow is super low level. You can use it for building AI pipelines or simple whiteboard apps. What are you trying to achieve with this poll? It doesn't make any sense to me.
